etc/kayobe/kolla/globals.yml

```diff
@@ -9,12 +9,12 @@ enable_docker_repo: {% raw %}"{{ 'overcloud' not in group_names or ansible_facts
 
 {% if kolla_base_distro == 'centos' %}
 bifrost_tag: yoga-20230217T160618
-neutron_tag: yoga-20230309T123152
+neutron_tag: yoga-20230315T105933
 {% elif kolla_base_distro == 'rocky' %}
 bifrost_tag: yoga-20230310T194732
```

Comment:
Missing rocky 9 image?

Reply:
Oh boy, now rocky - let me build it - do we have a way of building all three at the same time with the same tag? that would probably ease some pain next time ;-)

Reply:
The tag is generated during build so I don't think we have this capability yet 😞 At least you can easily trigger the build from the CLI with (run it three times while changing `distro`):

```
gh workflow run stackhpc-container-image-build.yml -F distro="<centos|rocky|ubuntu>" -F regexes="neutron" -r stackhpc/yoga
```

Reply:
<insert response about next-gen container image build automation>

Reply:
Done